src/components/pill-selector-dropdown/AvatarPill.js

```diff
@@ -0,0 +1,75 @@
+// @flow
+import React from 'react';
+import noop from 'lodash/noop';
+import classNames from 'classnames';
+import X from '../../icon/fill/X16';
+// $FlowFixMe this imports from a typescript file
+import LabelPill from '../label-pill';
+import Avatar from '../avatar';
+
+import './AvatarPill.scss';
+
+type Props = {
+    avatarUrl?: string,
+    className?: string,
+    hasWarning?: boolean,
+    id?: string | number,
+    isDisabled?: boolean,
+    isExternal?: boolean,
+    isSelected?: boolean,
+    isValid?: boolean,
+    onRemove: () => any,
+    showAvatar?: boolean,
```

Comment:
We are merging two concept with this one component: the new pill design, and an avatar pill which is a variation of that new pill. We should name the pill such that non-contact forms dont require this misleading name.

- BdlPill?
- RoundPill?
- etc.

We will eventually delete Pill.js, replacing it fully with the non-avatar pill component.

Reply:
I'll do Round Pill.